### ISSUE TITLE
NTR: PLOD1 enzyme activity low in skin fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8218]
New term request PLOD1 enzyme activity low in skin fibroblasts (elthat_101012011755): Lysyl hydroxylase 1  UMLS: 1857026

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000826
- label: Reduced procollagen-lysine,2-oxoglutarate 5-dioxygenase 1 activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:2504907
- def: Concentration or activity of procollagen-lysine,2-oxoglutarate 5-dioxygenase 1 (PLOD1; also known as Lysyl hydroxylase; EC 1.14.11.4) as measured in cultured fibroblasts is below the limits of normal. PLOD1 catalyzes the formation of hydroxylysine in collagens and other proteins with collagen-like amino acid sequences, by the hydroxylation of lysine residues in X-lys-gly sequences.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8218
- synonyms:
